ocw/lib/azure.py

```diff
@@ -9,12 +9,19 @@
 import re
 import time
 import logging
+from typing import Dict
 
 logger = logging.getLogger(__name__)
 
 
 class Azure(Provider):
-    __instances = dict()
+    __instances : Dict[str, "Azure"] = dict()
```

Comment:
Why not 
```suggestion
    __instances : Dict[str, Azure] = dict()
```

Reply:
or
```suggestion
    __instances : Dict[str, Type[Provider] = dict()
```

Reply:
I didn't found that `"Azure"` hint with google, thats why I'm asking :)

Reply:
`__instances : Dict[str, Type[Provider]] = dict()` 
this is not what I want to because this will allow also to add other siblings of Azure in this array 

` __instances : Dict[str, Azure] = dict()` 
this is not working and throwing some error . I asked in `python` channel for solution and get suggestion to put into a string . see https://chat.suse.de/channel/python?msg=dv8HPSDr7D5ihwLLg